docs/releases/upgrading/upgrade-v10.0.md

```diff
@@ -0,0 +1,110 @@
+# Upgrading from v9.0 to v10.0
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v9.0, please make sure that this is given!
+
+If unsure, run `invenio-cli install` from inside the instance directory before executing the listed steps.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older versions"
+
+    In case you have an InvenioRDM installation older than v9.0, you can gradually upgrade using the existing instructions to v9.0 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli`, for InvenioRDM v10 the release is v1.0.5
```

Comment:
make sure you point to the latest `invenio-cli` when we merge this. We haven't released yet the cli version for rdm-v10!

Reply:
Sure, but then I need to be notified when we are going to merge it.

Reply:
maybe replace v1.0.5 by TODO so we are aware and is easier to grep/spot. In any case it should be 1.1.0 (Since we are changing the defaul init branch).